frankenphp.go

```diff
@@ -501,6 +501,72 @@ func ServeHTTP(responseWriter http.ResponseWriter, request *http.Request) error
 	return nil
 }
 
+//export go_putenv
+func go_putenv(str *C.char, length C.int) C.int {
+	// Create a byte slice from C string with a specified length
+	s := C.GoBytes(unsafe.Pointer(str), length)
+
+	// Convert byte slice to string
+	envString := string(s)
+
+	// Check if '=' is present in the string
+	if idx := strings.IndexByte(envString, '='); idx != -1 {
+		// '=' found, set the environment variable
+		key := envString[:idx]
+		val := envString[idx+1:]
+
+		err := os.Setenv(key, val)
+		if err != nil {
+			return C.int(0) // Failure
+		}
+	} else {
+		// No '=', unset the environment variable
+		err := os.Unsetenv(envString)
+		if err != nil {
+			return C.int(0) // Failure
+		}
+	}
+
+	return C.int(1) // Success
+}
+
+//export go_getenv
+func go_getenv(name *C.char, length C.int, value **C.char, value_len *C.int) C.int {
+	if name == nil {
+		// Get all environment variables
+		env := os.Environ()
+		// Concatenate them with null separators
+		concatenatedEnv := strings.Join(env, "\x00") + "\x00" // Add an extra null terminator at the end
```

Comment:
You could maybe use our "go_string" type directly to prevent some useless copies and simplify the code? https://github.com/dunglas/frankenphp/blob/main/frankenphp.h#L14-L17